### DISCUSSION TITLE
PerceptualHash and distancing

### DISCUSSION THREAD

**Discussion by duckheadsoftware** [dlemstra/Magick.NET#848] [Help]
I'm looking to compare two photos to see if they're the same, or nearly the same.

I currently use my own code to do this, but I was wondering if there is a better way using Magick.net ?

I'm sure it probably has something to do with PerceptualHash, and I also assume that the image would need to be manipulated before calling this. Then of course there is "how to calculate the distance between two hashes"?

Thanks

**Comment by dlemstra** [comment]
You probably want to use the `Compare` method of `MagickImage` and check the return value. You can find some info about comparing here: https://legacy.imagemagick.org/Usage/compare/.
